WcaOnRails/app/views/regulations/scrambles/index.html.erb

```diff
@@ -24,6 +24,7 @@
       <li>If you are displaying scrambles on a digital device, only share <code>[Competition Name] - Computer Display PDFs.zip</code>.</li>
       <ul>
         <li>Make sure that only Delegates have access to any other files.</li>
+        <li>If you are using a Delegate's computer, use the operating system to create a guest user account and share only the passcode-protected computer display PDFs with it.</li>
```

Comment:
I think the recommendation should be to _always_ use a guest account, regardless if it's a delegate's computer or not. After all, the related security issues apply to any computer, not just that of a delegate. (I created a separate (passwordless) account on my laptop for scoretaking, so anyone can take over without me having to worry they get access to other parts of the machine)

Reply:
I'm mainly concerned about the Delegate's email or other TNoodle files.

The advice on this page is purpose normative (no "should"), so I would prefer to avoid requirements on this page if they're not needed when others bring their laptop.

Reply:
It is worth emphasizing here that this only applies to using the delegate's computer for viewing scrambles?

The purpose of this rule is kind of hard to discern from the rule itself. I think it would be good to put this bit about "the delegate's email or other TNoodle files" **somewhere**. Amending your commit message could be good enough.

Reply:
I'm going to keep this on the simple side for now. I'm certain we'll have to use other communication channels to reiterate this.